README.md

```diff
@@ -6,13 +6,13 @@ This project is based on the AddressBook-Level3 project created by the [SE-EDU i
 
 * This is the CS2103T Project for **Team CS2103T-W10-2**, the name of the project is StoreMando.
 
-* This product is a centralized inventory manager that helps to track the location of items in a household, their
-  individual quantities and respective expiry dates (if applicable). <br>
-  The usage of this product can eventually extend to other places outside households, however the core of our
-  development will surround the usage in a household.
-
-* It is named `StoreMando`, a combination of the words Storage and Commando, the goal of our product is to make it the
-  go-to product to track inventory.
+* StoreMando is an integrated platform fully customised for residents in households, residential colleges and halls,
+  with the aim of helping you manage your items effectively and efficiently. StoreMando allows you to keep track of your
```

Comment:
Should we be writing this as "with the aim of helping users manage their items effectively and efficiently". Since this is a more description of the project, I think using the same tone as user guide would not be appropriate. What do you think?

Reply:
Understand, will make changes on that! Thanks!